src/renderer/utils/random.ts

```diff
@@ -0,0 +1,3 @@
+export function generateRandomUUID() {
+  return crypto.randomUUID()
+}
```

Comment:
Why cryptographically secure UUIDs are used here?

Reply:
Good question! I don't really need the cryptographically secure randomness here but it seemed an easy way to get a random enough string without a lot of effort (vs. building a counter logic or my own way to use Math.random())

Reply:
I believe there is a UUID package we can use.

Reply:
please don't add new packages unless they are really necessary.
more packages bring more dependencies and with more dependencies grows the risk of getting an infected/vulnerable one. Not to mention build/install time and download size.

If there is a browser api directly for this it will likely be faster than a js implementation of the uuid function: https://dev.to/galkin/crypto-randomuuid-vs-uuid-v4-47i5

Reply:
yeah it might be faster but we are using crypto api which is not needed in this case. Doesn't a simple random large enough int suffice?

Reply:
I think it wants a string here. if you a really into it you can benchmark what is faster, but then again it does not make a difference because this is about opening dialogs, a code path that is not called that often. Also the crypto api is available at all browser targets we want to support, so I see no problem with using it.
If that should become a problem in the future, then we can still change it later.